modules/imgproc/src/canny.cpp

```diff
@@ -805,6 +865,20 @@ void Canny( InputArray _src, OutputArray _dst,
 
     Mat src = _src.getMat(), dst = _dst.getMat();
 
+    CV_OVX_RUN(
+        false && /* disabling due to accuracy issues */
```

Comment:
Looks like to Khronos OpenVX sample implementation bug. Other OpenVX implementations may work better.
/cc @vpisarev